File: server/utils/command/command.go
package command

type Command uint8

const (
	POSITION        Command = iota // 0
	MOVE                           // 1
	POSITION_RTT                   // 2
	MOVE_RTT                       // 3
	DEFAULT_RTT                    // 4
	USER_ASSIGNMENT                // 5
	PORT_REQUEST                   // 6
	PORT_ASSIGNMENT                // 7
)

File: client2/Scripts/Command/Command.cs
namespace Command;

public enum Command : byte
{
    POSITION = 0,
    MOVE = 1,
    POSITION_RTT = 2,
    MOVE_RTT = 3,
    DEFAULT_RTT = 4,
    USER_ASSIGNMENT = 5,
    PORT_REQUEST = 6,     // Changed to match server (was 7)
    PORT_ASSIGNMENT = 7,  // Changed to match server (was 6)
}
